assets/pages/digital-subscription-checkout/components/checkoutStage.jsx

```diff
@@ -23,8 +24,11 @@ import CheckoutForm from './checkoutForm';
 // ----- Types ----- //
 
 type PropTypes = {|
-  stage: Stage,
+  checkout: {
+    stage: Stage,
+  },
```

Comment:
Can `stage` live at the top level of the `PropTypes`?

Reply:
Possibly. But you suggested we put it into the checkout wrapper.